platform/core/src/classes/CommandsManager.ts

```diff
@@ -169,7 +169,7 @@ export class CommandsManager {
    *   the commandOptions specified in the base.
    */
   public run(
-    toRun: Command | Commands | Command[] | string | undefined,
+    toRun: Command | Commands | (Command | string)[] | string | undefined,
```

Comment:
could we please update the docsring above and give example of what we can handle 

something like 

run ('updateMeasurement') 
run (['updateMeasurement', 'displayWhatever'])
// other scnearios